code/src/index.css

```diff
@@ -1,13 +1,206 @@
+*{
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: linear-gradient(180deg, rosybrown 60%, rgb(155, 240, 240));
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.mainContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.surveyContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
For the whole CSS, nice and neat code. Nicely done with the media queries, even if the styling might work better for a mobile screen, the text changes size so it is easy to read even on a larger screen. Well done!